STYLEGUIDE.md

```diff
@@ -295,6 +298,12 @@ Joined as a noun/modifier, separated as a phrasal verb.
 ### Services
 Use proper case.
 
+### spaces after a period. one, not two (or three or four)
+Contrary to what you may have been taught, most current style guides dictate the use of **one** space after a period, not two.
```

Comment:
```suggestion
Use only one space after a period, not two.
```

Reply:
People are using the style guide for quick reference. I don't think this point is widely disputed, but if it is, we'll block attempts to change it.